app/models/search_vector.rb

```diff
@@ -1,8 +1,76 @@
 class SearchVector < ActiveRecord::Base
+
+  WEIGHT_VALUES = [
```

Comment:
These are the weights applied to different sections of the discussion. For example, a discussion title is ranked A (1.0), while a motion title is ranked B (0.3), a comment body is ranked D (0.03), etc. They're pulled out here so they can be easily tweaked (maybe even as an ENV?)